src/ArraysLength.java

```diff
@@ -0,0 +1,51 @@
+import java.util.Scanner;
+
+public class ArraysLength {
+    public static void main(String[] args) {
+        Scanner myObj = new Scanner(System.in);
+        System.out.print("Please enter the length of array : ");
+        int userinput = myObj.nextInt();
+        userinput=userinput>0? userinput:0;
+
+        int[] arrOfnumbers;
+
+        switch (userinput) {
+            case 0:
+                System.out.println("Please enter positiv number!!!");
```

Comment:
Why does your system continue to work normally? Your program should be terminated at this point since 0 input for array length is invalid.